website/client/js/services/socialServices.js

```diff
@@ -34,8 +36,27 @@
       }
     }
 
+    hello.init({
+      facebook : window.env.FACEBOOK_KEY,
+      google : window.env.GOOGLE_CLIENT_ID
+    });
+
+    function socialLogin(network){
+      hello(network).login({scope:['email', 'profile']}).then(function(auth){
+        $http.post(ApiUrl.get() + "/api/v3/user/auth/social", auth)
+          .success(function(res, status, headers, config) {
+            runAuth(res.data.id, res.data.apiToken);
+          }).error(errorAlert);
```

Comment:
Oh. That's only defined in authCtrl.js. so it's not actually accessible here. What would the best way be to make that accessible in both places?